app/models/opportunity_details/via_voucher.rb

```diff
@@ -7,7 +7,43 @@ class ViaVoucher < Base
     def unit_name
       voucher.unit.try :name
     end
-    
+
+    def unit_on_ground_floor?
```

Comment:
These should all be defined in the base class and return nil.  Since these require a unit they should be re-defined in `app/models/opportunity_details/via_unit.rb` where you won't need `return nil unless voucher.unit`